### PR TITLE
feat(*): support clause parsing

### PR DESCRIPTION
**Pull request by irmerk**
# No Issue
Support parsing, most notably in the demo

### Changes
- Maintain the debounce for parsing
- Setup redux store for parsing in the demo

### Flags
- `cicero-ui` does not parse on it's own, the parent component / consumer will need to do this

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,8 @@
     "node-sass": "^4.13.1",
     "prop-types": "^15.7.2",
     "ramda": "^0.26.1",
+    "react-redux": "^7.2.0",
+    "redux": "^4.0.5",
```
Comment (DianaLease) [inline package.json:34]: Would it be possible to make these dev dependencies since Redux is only used in the demo?

**Comment by jeromesimeon** [comment]
> I'm getting parse exceptions locally. Not sure if I'm building it wrong. I did `npm install` `npm run dev`
> 
> ```
>   ``` <clause src="https://templates.accordproject.org/archives/acceptance-of-delivery@0.13.1.cta" clauseid="345"/>
>   ^
> Unexpected "`"
> ```

Maybe we need something similar to what we used to do in the demo for the `slate47` version?
https://github.com/accordproject/cicero-ui/blob/cb874335ba6e507249fa6cc39c1120067edf84df/demo/src/index.js#L92-L97

i.e., get the content of the clause node rather than just the clause node?

File: src/stories/Demo/Demo.stories.js [hunk 2]
```diff
@@ -210,6 +210,7 @@ export const demo = () => {
         readOnly={readOnly}
         ref={refUse}
         // loadTemplateObject={fetchTemplateObj}
+        onClauseUpdated={() => {console.log('Demo')})}
```
Comment (DianaLease) [inline src/stories/Demo/Demo.stories.js:213]: There's an extra `)` between the curlies

Comment (irmerk) [reply src/stories/Demo/Demo.stories.js]: 👀

**Pull request merged**